### ISSUE TITLE
Instant combat if you send your pet to attack

### ISSUE THREAD

**Issue by RhythmOfficial** [Atlantiss/BugTracker#6521]
This is bugged and extremely annoying in arena.
Basically what happens is, when you send your pet in to attack your target the person who send the pet in to attack gets into combat instantly when they send their pet in.

This was bugged on retail and got hot fixed a bit later arround patch 4.2.2 or something like that.

Please fix this as this bug is annoying as hell.

**Comment by Kejmil** [comment]
proofs?

**Comment by Fearyourself** [comment]
This was indeed bugged on retail but it never got hotfixed. It was like this all the way throughout cataclysm.

**Comment by RhythmOfficial** [comment]
It is simple game mechanics, and it did actually get fixed during cataclysm.
And even if it didn't get fixed it is still a bug and therefore has to be fixed, or are you going to replicate retail bugs on a private server aswell because "retail had it the exact same" retail was NOT bug free either.
As in a matter of fact there were game breaking bugs that didn't get fixed untill fucking patch 4.2

Anyway here is your proof : https://youtu.be/vZhu-G4riyo?t=38m44s

Watch how he is trying to get combat against the RLS, he sends his pet in but doesn't get combat instantly his pet is running for a couple yards but he only receives combat once he curses one of the shamans totems and not from his pet running in while attacking.

FIX THIS BUG!

**Comment by artamedan** [comment]
Pet is a part of your attack, its damage counts on you so it is good and obvious that you get into combat the moment your pet is attacking.
Anyway, be happy, that pets are so bugged here that they attack without LoS, they pass through textures, they attack still when hunter is dead, etc.
And don't take playing on private server so seriously- it is only an emulator, not real game. Will never work like it should.

**Comment by RhythmOfficial** [comment]
@artamedan 
Can you even read?
Don't comment if you're not going to say anything smart, you just litterally said nothing about what I talked about in this report.

Everyone knows if your pet attacks something that you get combat from it.
That is not what this fucking report is about, read then speak please. 
God

**Comment by Fearyourself** [comment]
Why are you so angry? Even if it's bugged on retail it's supposed to be bugged on atlantiss too. That's how they do thing around here. Besides, it was bugged on retail and will be on atlantiss aswell. Anger checks out

**Comment by RhythmOfficial** [comment]
@Fearyourself 
Yeah please drink some more of that polish vodka and drink yourself to dead please.
People with an attitude and mindset like you just fucking piss me off thats why so now please go be a pleb somewhere else.

**Comment by Fearyourself** [comment]
I'm sorry about your girlfriend breaking up with you man. Hang in there alright homie
